dev/ToastNotifications/NotificationTransientProperties.cpp

```diff
@@ -22,7 +22,7 @@ NotificationTransientProperties::NotificationTransientProperties(winrt::ToastNot
 {
     m_suppressPopup = toastNotification.SuppressDisplay();
 
-    if (toastNotification.Priority() == winrt::Microsoft::Windows::ToastNotifications::ToastPriority::High)
+    if (toastNotification.Priority() == winrt::ToastPriority::High)
     {
```

Comment:
General comment: Please merge with the latest renaming changes.

Reply:
PR is in-sync with the l1 and has no merge conflicts at this time